ezbake/config/conf.d/ca.conf

```diff
@@ -7,4 +7,7 @@ certificate-authority: {
 
     # enable the separate CRL for Puppet infrastructure nodes
     # enable-infra-crl: false
+    #allow-auto-renewal: false
```

Comment:
This file gets installed, can you add some explanation comments above each entry so users will understand what they are for?  I can help with the wording if that would be helpful.